libs/external-router/driver/magento/src/model/magento-url-resolver.ts

```diff
@@ -0,0 +1,5 @@
+import { MagentoUrlResolution } from './resolution';
+
+export interface MagentoUrlResolutionResponse {
+  urlResolver: MagentoUrlResolution;
```

Comment:
I think this should be `urlResolution` to keep the naming consistent

Reply:
This is the key from GraphQl in v2.4, it can't be changed. 230ac7be